src/Symfony/Resources/views/fieldtype/field_view.html.twig

```diff
@@ -4,7 +4,8 @@
 {{ render(controller(
     "ez_content:viewAction",
     {
-        "contentId": contentInfo.id,
+        "content": content,
+        "location": location is defined ? location : null,
```

Comment:
You can use null-coalescing operator instead: https://twig.symfony.com/doc/3.x/templates.html#other-operators

```suggestion
        "location": location ?? null,
```

Reply:
Good idea, thank you. I'm getting too old for those fancy syntaxic candies 👴